charts/coatrack/values.yaml

```diff
@@ -0,0 +1,159 @@
+# Default values for coatrack
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+nameOverride: ""
+fullnameOverride: ""
+
+imageRepo: ubuntu
+imageTag: latest
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
```

Comment:
Doesn't that mean that by default, nothing is deployed? I'm a little confused, whats the prupose of this chart? It does not deploy anything coatrack related.